### PR TITLE
add service monitor for postgresql

### PR DESCRIPTION
**Pull request by nek-00-ken**
> Please remove all the lines starting with '>'
> Hi there! Thanks for submitting a pull request to the Concourse Helm chart!
> If you haven't already, please take a look at our [Code of Conduct](https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md).

> To help us review your PR, please fill in the following information.
> Feel free to delete any sections not applicable to your pull request.

# Existing Issue
> Is there an existing issue related to this PR? Fill in the issue number as follows: Fixes #1234.

Fixes # .

# Why do we need this PR?
> No issue number? Please give us a few lines of context describing the need for this PR.

# Changes proposed in this pull request
> What are the changes included in this PR? Feel free to add as many lines as needed below.

* 
* 
*

# Contributor Checklist
> Are the following items included as part of this PR? Please delete checkbox items that don't apply.
- [ ] Variables are documented in the `README.md`

# Reviewer Checklist
> This section is intended for the core maintainers only, to track review progress. Please do not
> fill out this section.
- [ ] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -375,6 +375,14 @@ concourse:
       ##
       database: atc
 
+      ## Postgresl service monitor (optional, default: true)
```
Comment (lightcode) [inline values.yaml]: Small typo: `Postgresql`

File: values.yaml [hunk 2]
```diff
@@ -375,6 +375,14 @@ concourse:
...
+      ##
+      serviceMonitor:
+        enabled: true
+        interval: 30s
+        labels: {}
+        relabelings:
```
Comment (lightcode) [inline values.yaml]: I think these options are related to postgres-operator only. Can you move it under `operator`?

Like this:

```
concourse.web.postgres.serviceMonitor -> concourse.web.postgres.operator.serviceMonitor
```

File: templates/postgresql-service-monitor.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.concourse.web.postgres.serviceMonitor.enabled }}
```
Comment (lightcode) [inline templates/postgresql-service-monitor.yaml]:
```suggestion
{{- if and .Values.concourse.web.postgres.operator.enabled .Values.concourse.web.postgres.operator.serviceMonitor.enabled }}
```

**Review by lightcode — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 2 above)
Comment (nek-00-ken) [reply values.yaml]: done

File: templates/postgresql-service-monitor.yaml (hunk 3 above)
Comment (nek-00-ken) [reply templates/postgresql-service-monitor.yaml]: done

File: values.yaml (hunk 1 above)
Comment (nek-00-ken) [reply values.yaml]: done

File: templates/postgresql-service-monitor.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+{{ toYaml . | trim | indent 4 }}
+    {{- end }}
+spec:
+  endpoints:
+  - interval: {{ .Values.concourse.web.postgres.operator.serviceMonitor.interval }}
+    path: /metrics
+    port: exporter
+    {{- if .Values.concourse.web.postgres.operator.serviceMonitor.relabelings }}
+    relabelings:
+{{ toYaml .Values.concourse.web.postgres.operator.serviceMonitor.relabelings | nindent 6 }}
```
Comment (lightcode) [inline templates/postgresql-service-monitor.yaml]: The function `nindent` adds a new line before rendering the text. You can write everything on the same line:

```suggestion
    relabelings: {{ toYaml .Values.concourse.web.postgres.operator.serviceMonitor.relabelings | nindent 6 }}
```

**Review by lightcode — CHANGES_REQUESTED** [review]

**Review by lightcode — APPROVED** [review]

**Pull request merged**